### PR TITLE
Make some updates to runtime for easier reuse

### PR DESCRIPTION
**Pull request by ilblackdragon**
Make runtime process_* public, add separate set_genesis_hash to storage.
Updated pynear signed_transaction_pb2 file with new protos.

### CODE REVIEW

File: node/runtime/src/lib.rs [hunk 1]
```diff
@@ -710,7 +710,7 @@ impl Runtime {
         debug!(target: "runtime", "{}", log_str);
     }
 
-    fn process_transaction(
+    pub fn process_transaction(
```
Comment (bowenwang1996) [inline node/runtime/src/lib.rs]: Why making this public?

File: core/storage/src/storages/mod.rs [hunk 2]
```diff
@@ -195,6 +207,19 @@ where
         )
     }
 
+    #[inline]
+    pub fn set_best_block_index(&mut self, value: u64) -> io::Result<()> {
+        let mut key = self.enc_slice(self.genesis_hash.expect(MISSING_GENESIS_ERR).as_ref());
+        key.extend_from_slice(&[0]);
```
Comment (bowenwang1996) [inline core/storage/src/storages/mod.rs]: Why this is line necessary?

File: node/runtime/src/lib.rs (hunk 1 above)
Comment (ilblackdragon) [reply node/runtime/src/lib.rs]: To reuse runtime when need to process one tx at a time.

File: core/storage/src/storages/mod.rs (hunk 2 above)
Comment (ilblackdragon) [reply core/storage/src/storages/mod.rs]: I'm using the same column for best hash and best index, so storing index at `genesis_hash` + [0]

Comment (bowenwang1996) [reply core/storage/src/storages/mod.rs]: I see. Why not making a separate column?

Comment (ilblackdragon) [reply core/storage/src/storages/mod.rs]: Just feels weird having columns with 1 value in them. Generally the design of how we storing this values seems imperfect - we have a hashmap of genesis hash to best_hash <-- e.g. there always going to be one value there.

Comment (nearmax) [reply core/storage/src/storages/mod.rs]: That's just a limitation of a columnio storage. If we want to store some miscellaneous one-off data like genesis and best index we can keep them in a single meta column.

**Pull request merged**